### PR TITLE
Add React to an Existing Project page translated to Telugu

### LINKED ISSUE #10: Add React to an Existing Project

### PR DESCRIPTION
**Pull request by srikanth-kandi**
Hello @g-savitha, @gsteja2307 and @vamsikrishnareddybevara

I have translated the entire page of [Add React to an Existing Project](https://te-react-dev.vercel.app/learn/add-react-to-an-existing-project) to Telugu by following the [Telugu Glossary](https://github.com/reactjs/te.react.dev/issues/13) and other guidelines provided.

Please carefully review the translated content, and merge it ASAP because I'm eager to translate other pages also but as mentioned in #1 "**Please take only one page at a time**".

Merging this PR will close #10

Thank you :)

### CODE REVIEW

**Comment by srikanth-kandi** [comment]
Please add me as one of the maintainers, I have experience with React and have completed many projects using it.
Check them on my profile @srikanth-kandi and my certifications mentioned in [LinkedIn](https://www.linkedin.com/in/srikanthkandi/details/certifications/)

@g-savitha @gsteja2307

According to the guidelines about **Adding a Maintainer** mentioned in [translations.react.dev](https://github.com/reactjs/translations.react.dev?tab=readme-ov-file#adding-a-maintainer), if any of the maintainers like @vamsikrishnareddybevara isn't active on this repository for the last 3 months or so. They need to be replaced by others.

So I hope you consider adding me as another maintainer for this repository.

**Comment by srikanth-kandi** [comment]
Hello @g-savitha please review this pull request, as I'm also working on another [page](https://github.com/reactjs/te.react.dev/issues/9), so if PR was merged I can send another pull request of that translated page.

File: src/content/learn/add-react-to-an-existing-project.md
Comment (gsteja2307) [inline src/content/learn/add-react-to-an-existing-project.md:13]: మీరు React ను లోకల్ గా వాడాలి అనుకొంటే [Node.js](https://nodejs.org/en/) ను ఇన్స్టాల్ చేసుకోవలసి ఉంటుంది లేదా ** మీరు [React ను ఆన్‌లైన్‌లో ప్రయత్నించండి ](/learn/installation#try-react) లేదా సాధారణ HTML పేజీతో ప్రయత్నించవచ్చు,   కానీ వాస్తవానికి మీరు ప్రాజెక్ట్ డెవలప్మెంట్కు  వాడాలనుకునే  JavaScript టూలింగ్ కు Node.js అవసరం

File: src/content/learn/add-react-to-an-existing-project.md
Comment (gsteja2307) [inline src/content/learn/add-react-to-an-existing-project.md:7]: మీరు ఇప్పటికే ఉన్న మీ ప్రాజెక్ట్‌కి కొంత ఇంటరాక్టివిటీని జోడించాలనుకుంటే, దాన్ని React లో మళ్ళీ రాసే అవసరం లేకుండా, మీ ఉన్న స్టాక్‌లో React ను జోడించి. ఇంటరాక్టివ్ React కాంపోనెంట్లను మీరు ఎక్కడైనా రెండర్ చేయవచ్చు.

**Comment by gsteja2307** [comment]
thanks for your effort on translating the project to Telugu. I noticed some sentences seem to be directly translated, which may not align with the contextual meaning in Telugu. Could you please review the translation for grammatical accuracy and context sensitivity? It's important that the translated content maintains the original message and flows naturally in Telugu. A thorough proofreading would be greatly appreciated. Thanks!

> Hello @g-savitha, @gsteja2307 and @vamsikrishnareddybevara
> 
> I have translated the entire page of [Add React to an Existing Project](https://te-react-dev.vercel.app/learn/add-react-to-an-existing-project) to Telugu by following the [Telugu Glossary](https://github.com/reactjs/te.react.dev/issues/13) and other guidelines provided.
> 
> Please carefully review the translated content, and merge it ASAP because I'm eager to translate other pages also but as mentioned in #1 "**Please take only one page at a time**".
> 
> Merging this PR will close #10
> 
> Thank you :)

Appreciate your work on the Telugu translation of this page. I've noticed certain phrases seem directly translated, potentially losing the intended meaning in Telugu. Please recheck for grammatical and contextual accuracy. Ensuring that the translation accurately conveys the original message and reads naturally in Telugu is crucial. 

Detailed proofreading would be very helpful. Thanks!

I've provided some examples in the PR review. Please ensure a comprehensive check before finalizing your changes. Additionally, be cautious with text in square brackets, typically links, to preserve context. If feasible, also update the linked titles for clarity and relevance.

**Review by g-savitha — COMMENTED** [review]
Comments need not to be updated, as these will not be shown to end user.

File: src/content/learn/add-react-to-an-existing-project.md
Comment (g-savitha) [inline src/content/learn/add-react-to-an-existing-project.md:151]: మీ పేజీ మొత్తం React తో నిర్మించబడే వరకు క్రమంగా “పైకి Veluthu undandi"

**Review by g-savitha — CHANGES_REQUESTED** [review]

**Comment by srikanth-kandi** [comment]
> Comments need not to be updated, as these will not be shown to end user.

But as mentioned in this [style-guide](https://github.com/reactjs/translations.react.dev/blob/main/style-guide.md#text-in-code-blocks) **Leave text in code blocks untranslated except for comments**

So if we translate the comments of code blocks to Telugu it can help easily understand the purpose of that code block for a newbie.

@g-savitha

File: src/content/learn/add-react-to-an-existing-project.md
Comment (srikanth-kandi) [reply src/content/learn/add-react-to-an-existing-project.md:27]: please suggest the easy sentence, i didn't get one

**Comment by srikanth-kandi** [comment]
Can anyone review this PR

@g-savitha @gsteja2307

**Comment by srikanth-kandi** [comment]
> Please add me as one of the maintainers, I have experience with React and have completed many projects using it. Check them on my profile @srikanth-kandi and my certifications mentioned in [LinkedIn](https://www.linkedin.com/in/srikanthkandi/details/certifications/)
> 
> @g-savitha @gsteja2307
> 
> According to the guidelines about **Adding a Maintainer** mentioned in [translations.react.dev](https://github.com/reactjs/translations.react.dev?tab=readme-ov-file#adding-a-maintainer), if any of the maintainers like @vamsikrishnareddybevara isn't active on this repository for the last 3 months or so. They need to be replaced by others.
> 
> So I hope you consider adding me as another maintainer for this repository.

You guys didn't consider making me a maintainer, and you are also not responding to the contributions.
Consider responding to this repository updates by sparing at least some time a day 🙏

@g-savitha @gsteja2307

**Comment by srikanth-kandi** [comment]
Hey @g-savitha and @gsteja2307 

Please review this and my existing pull requests, I requested you make me a maintainer on #1. Otherwise, add some other maintainers you know who will actively monitor the contributions on this repository.

Waiting for your response.

**Pull request merged**
